pyvips/gvalue.py

```diff
@@ -219,7 +219,7 @@ def get(self):
             result = []
             for i in range(0, pint[0]):
                 vi = array[i]
-                vips_lib.g_object_ref(vi)
+                gobject_lib.g_object_ref(vi)
```

Comment:
If I use `vips_lib` here I see:
```
AttributeError: function/symbol 'g_object_ref' not found in library 'libvips-42.dll': error 0x7f
```

Perhaps this fix should also be applied to lua-vips, see [here](https://github.com/jcupitt/lua-vips/blob/2a3917a7d489bcbeaa324736e089d704d337ac0b/src/vips/gvalue.lua#L214).